apps/server/src/shared/infra/identity-management/keycloak/service/keycloak-identity-management-oauth.service.spec.ts

```diff
@@ -50,7 +50,8 @@ describe('KeycloakIdentityManagementService', () => {
 	});
 
 	afterEach(() => {
-		jest.clearAllMocks();
+		kcIdmOauthService.resetOauthConfigCache();
```

Comment:
@SevenWaysDP You also added it to all the setups. I would suggest to have it only in one place.